src/frontend/js/widgets/Dashboard/utils/dashboardRoutes.tsx

```diff
@@ -84,14 +61,10 @@ const getDashboardRouteAttribute =
       opts?: IntlMessageFormatOptions,
     ]
   ) => {
-    const isLabel = attribute === RouteAttributes.LABEL;
-
-    const message = isLabel ? DASHBOARD_ROUTE_LABELS[path] : DASHBOARD_ROUTE_PATHS[path];
-    return intl.formatMessage(message, ...options);
+    return intl.formatMessage(DASHBOARD_ROUTE_LABELS[path], ...options);
   };
 
-/** Get the provided dashboard route label in the active locale */
-export const getDashboardRouteLabel = getDashboardRouteAttribute(RouteAttributes.LABEL);
-
 /** Get the provided dashboard route path in the active locale */
-export const getDashboardRoutePath = getDashboardRouteAttribute(RouteAttributes.PATH);
+export const getDashboardRoutePath = (originalPath: string, params: any = {}) => {
```

Comment:
This function is just alias for `generatePath` and therefor not needed

Reply:
old commit here